### ISSUE TITLE
Add other methods to mcfly from reviewer

### ISSUE THREAD

**Issue by cwmeijer** [NLeSC/mcfly#207]
There are a few alternatives such as https://github.com/hfawaz/dl-4-tsc, https://github.com/sktime/sktime-dl, and https://github.com/hfawaz/InceptionTime.

Check out the links above and see if one of these contain a method or architecture that is easy to add to mcfly.

**Comment by florian-huber** [comment]
I have tried out the InceptionTime recently. And I also started to modify the code a bit, e.g. using tensorflow.keras instead of keras etc.
It would require some extra work to clean it up and make it part of mcfly, but probably not too much!

https://github.com/epodium/EEG_dyslexia_prediction/blob/master/inception_time.py

**Comment by florian-huber** [comment]
- [ ] Decide on which method to add (InceptionTime, Resnet, others?)
- [ ] Implement methods.
- [ ] Decide on hyperparameter space to be explored randomly by mcfly
- [ ] Decide on how to randomly draw models from all type categories (CNN, CNN+LSTM, inceptionTime....)

**Comment by florian-huber** [comment]
- I implemented InceptionTime and briefly checked that it indeed works (it gave good results on a test dataset, I will share the notebook soon).

- I changed the random model selection. So far it was just picking either CNN or LSTM with 50% chance. However, that felt weird when using 4 model types since often people would then not even have all model types in their say 10 models they generate. Now it will randomly draw one model type after another (without replacement) and then repeat. Example: If you ask for 12 models and if we provide 4 types (CNN, LSTM, InceptionTime, Resnet) it will give back 3 random models of each type.
